core/src/test/java/io/kestra/core/repositories/AbstractDashboardRepositoryTest.java

```diff
@@ -0,0 +1,140 @@
+package io.kestra.core.repositories;
+
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.dashboards.Dashboard;
+import io.kestra.core.models.dashboards.TimeWindow;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.services.ExecutionService;
+import io.micronaut.data.model.Pageable;
+import jakarta.inject.Inject;
+import org.codehaus.plexus.util.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@KestraTest
+public abstract class AbstractDashboardRepositoryTest {
+    public static final String NAMESPACE = "io.kestra.unittest";
+    public static final String TENANT_ID = "tenantId";
+    private static final String DASHBOARD_DESCRIPTION = "Sample dashboard";
+    @Inject
+    protected ExecutionService executionService;
+
+    @Inject
+    protected DashboardRepositoryInterface dashboardRepositoryInterface;
+    @Inject
+    protected RunContextFactory runContextFactory;
+
+    public static Dashboard dashBoardEntry(String id, String description) {
+        return Dashboard.builder()
+            .id(id)
+            .description(StringUtils.defaultString(description, "Default overview dashboard"))
```

Comment:
Move the default as it can lead to false-positive tests (updated dashboard not updated and just falling back to default value)
```suggestion
            .description(description)
```

Reply:
done!